sdk/compiler/damlc/tests/daml-test-files/HexString.daml

```diff
@@ -0,0 +1,95 @@
+-- Copyright (c) 2020, Digital Asset (Switzerland) GmbH and/or its affiliates.
+-- All rights reserved.
+
+-- @SUPPORTS-LF-FEATURE DAML_CCTP
+
+module HexString where
+
+import DA.Assert ((===))
+import DA.Crypto.Text
+import Daml.Script
+
+n3: Numeric 3
+n3 = 0.123
+
+n7: Numeric 7
+n7 = 0.12345_67
+
+n10: Numeric 10
+n10 = 0.12345_67890
+
+main =
+  script do
+    -- `HasToHex Int` and `HasFromHex Int`
+    toHex 0 === "00"
+    toHex 1 === "01"
+    toHex 15 === "0f"
+    toHex 16 === "10"
+    toHex 255 === "ff"
+    toHex 256 === "0100"
+    toHex 257 === "0101"
+    toHex 3735928559 === "deadbeef"
+    fromHex "" === 0
+    fromHex "00" === 0
+    fromHex "01" === 1
+    fromHex "0a" === 10
+    fromHex "0F" === 15
+    fromHex "10" === 16
+    fromHex "Ff" === 255
+    fromHex "0100" === 256
+    fromHex "0101" === 257
+    fromHex "DeadBeef" === 3735928559
+
+    -- `HasToHex Text` and `HasFromHex Text`
+    toHex "" === ""
+    toHex " " === "20"
+    toHex "a" === "61"
+    toHex "Hello World!" === "48656c6c6f20576f726c6421"
+    toHex "DeadBeef" === "4465616442656566"
+    fromHex "" === ""
+    fromHex "20" === " "
+    fromHex "61" === "a"
+    fromHex "48656c6c6f20576f726c6421" === "Hello World!"
+    fromHex "4465616442656566" === "DeadBeef"
```

Comment:
max and min bounds for the different int types.

Reply:
Could you expand on this please? I presume you mean for the `isBytes32Hex`, `isUInt32Hex`, etc. range of functions?

Reply:
yes